tests/test_make.py

```diff
@@ -1587,3 +1587,13 @@ class B(A):
             "Comparision of subclasses using %s is deprecated and will be "
             "removed in 2019." % (op,)
         ) == w.message.args[0]
+
+
+class TestDocs:
```

Comment:
Why do you think we need a new test class for this?

Reply:
I wasn't sure on the policy you use to separate test classes but usually I create test classes around a concern (in this case the docs). If at the end of the PR I see that there won't be enough tests to justify a dedicated class, I will add them to an existing class.